test/api_searchProducts_test.dart

```diff
@@ -142,6 +142,29 @@ void main() {
       expect(result.count, greaterThan(900));
     });
 
+    test('search products by keywords 2', () async {
```

Comment:
I would mention the issue: if you don't know the context you may think that this test is not very interesting.